### PR TITLE
issue-24: Combined RouterConifg in one file

### LINKED ISSUE #24: Organize one file of BrowserRouter
I will make only one file to summarize all url link. 
Please you guys update each link whenever you want to link it.

### PR DESCRIPTION
**Pull request by eunbeek**
Please check this file and let me ask it if you guys not understand what it is going on

### CODE REVIEW

File: src/RouterConfig.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    render() {
+        return (
+            <BrowserRouter>
+            <Switch>
+              <Route exact path='/' render={() =>
+                  <header className="App-basic">
+                      <p>
+                        Body Contouring Clinic <br/>
+                        React : true <br/>
+                        {`Express : ${this.state.backServer}`}
```
Comment (yuanLeeMidori) [inline src/RouterConfig.js]: This causes error.
`this.state` is not set in this file. It is still in the App.js file

File: src/RouterConfig.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem, FormGroup, Form, FormControl, Grid, Row, Col, Button } from 'react-bootstrap'
+import { Link, Switch, Redirect, Route, BrowserRouter } from 'react-router-dom'
+import { LinkContainer } from 'react-router-bootstrap'
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css'  // temporary, we can create our css later
+import AppointmentHome from './Appointment/AppointmentHome'
+import ServiceHome from './Service/ServiceHome'
+import RequestHome from './Request/index'
```
Comment (yuanLeeMidori) [inline src/RouterConfig.js]: `'./Request/index'` export a component called `index`, not `RequestHome`

File: src/RouterConfig.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import CreateRequest from './Request/CreateRequest';
```
Comment (yuanLeeMidori) [inline src/RouterConfig.js:10]: There is no CreateRequest file or component

**Review by yuanLeeMidori — CHANGES_REQUESTED** [review]

File: src/RouterConfig.js (hunk 1 above)
Comment (eunbeek) [reply src/RouterConfig.js]: Yeah, I was fixing this one. I will change it tomorrow morning.

File: src/RouterConfig.js (hunk 2 above)
Comment (eunbeek) [reply src/RouterConfig.js]: In my part, I made index file to call both RequestHome and  SideBar so I called it. However, It makes some errors when I call another page under Request directory, not defined in index, It does not contain SideBar in the render. 

I need to think it more .

File: src/RouterConfig.js (hunk 3 above)
Comment (eunbeek) [reply src/RouterConfig.js:10]: Right, I did not request new PR for this :) 

Will be soon updated.

**Review by yuanLeeMidori — APPROVED** [review]

**Review by rko7 — APPROVED** [review]

**Pull request merged**